### ISSUE TITLE
Compilation error Mirror-core 1.0.0 package, Coq 8.5.0

### ISSUE THREAD

**Issue by clarus** [gmalecha/mirror-core#106]
Hi,

I have got a compilation error with the package `coq-mirror-core.1.0.0`:
* full output: https://coq-bench.github.io/clean/Linux-x86_64-4.02.3-2.0.1/released/8.5.0/mirror-core/1.0.0.html
* extract of the trace:
```
- "coqc"  -q  -Q "theories" MirrorCore -Q "examples" McExamples -I "src" -I "/home/bench/.opam/ocaml-base-compiler.4.02.3/lib/coq//user-contrib/PluginUtils"   examples/Cancel/Monoid
- "coqc"  -q  -Q "theories" MirrorCore -Q "examples" McExamples -I "src" -I "/home/bench/.opam/ocaml-base-compiler.4.02.3/lib/coq//user-contrib/PluginUtils"   examples/Hoare/ILogic
- "coqc"  -q  -Q "theories" MirrorCore -Q "examples" McExamples -I "src" -I "/home/bench/.opam/ocaml-base-compiler.4.02.3/lib/coq//user-contrib/PluginUtils"   examples/AutoRewrite/DemoQuantPullLtac
- "coqc"  -q  -Q "theories" MirrorCore -Q "examples" McExamples -I "src" -I "/home/bench/.opam/ocaml-base-compiler.4.02.3/lib/coq//user-contrib/PluginUtils"   theories/TypesI
- File "src/reify_Lambda_shell.ml4", line 327, characters 6-138:
- Error: Unbound module Ephemeron
- Makefile.coq:495: recipe for target 'src/reify_Lambda_shell.cmo' failed
- make[1]: *** [src/reify_Lambda_shell.cmo] Error 2
- make[1]: *** Waiting for unfinished jobs....
- File "src/reify_Lambda_shell.ml4", line 327, characters 6-138:
- Error: Unbound module Ephemeron
- Makefile.coq:498: recipe for target 'src/reify_Lambda_shell.cmx' failed
- make[1]: *** [src/reify_Lambda_shell.cmx] Error 2
```

After some investigations I though it was because of this commit in Coq: https://github.com/coq/coq/commit/b98e4857a13a4014c65882af5321ebdb09f41890 Actually this commit never made it into 8.5.0 and it seems due to this [patch file](https://github.com/ocaml/opam-repository/blob/master/packages/coq/coq.8.5.0/files/ephemeron-rename.patch) in the Coq package:
![image](https://user-images.githubusercontent.com/316665/61367850-cb3a6380-a88c-11e9-9146-e784b287d82c.png) To me the fault is from this patch.

What do you think we should do? Remove the Coq patch (maybe too late)? Remove this package version? Is there an other way to install this version?

**Comment by gmalecha** [comment]
I can update mirror-core to be compatible with the patch if is just a simple renaming.

**Comment by clarus** [comment]
Ah, OK, but this is about an old version (`1.0.0`) and is already fixed in the next one (`1.0.1`). The question is more about the precise `coq-mirror-core.1.0.0` package which is available on the repo but cannot install as far as I understand.

**Comment by clarus** [comment]
I think the simplest is to remove it.

**Comment by gmalecha** [comment]
Oh, sounds like a better idea if it is already fixed in 1.0.1. Do I need to do anything to remove it?

**Comment by clarus** [comment]
No, I can do it on the repo. This was to confirm. Thanks!
